polkadot/node/network/collator-protocol/src/validator_side_experimental/peer_manager/backend.rs

```diff
@@ -24,7 +24,7 @@ use std::collections::{BTreeMap, BTreeSet, HashMap};
 #[async_trait]
 pub trait Backend {
 	/// Instantiate a new backend.
-	async fn new() -> Self;
+	async fn new(stored_limit_per_para: u8) -> Self;
```

Comment:
I think you can remove `new` from the trait.  How the backend is constructed should be implementation specific.